api/src/main/java/org/openmrs/api/impl/UserServiceImpl.java

```diff
@@ -731,7 +731,14 @@ public User setUserActivationKey(User user) throws MessageException {
 		AdministrationService adminService = Context.getAdministrationService();
 		Locale locale = getDefaultLocaleForUser(user);
 		
-		String link = adminService.getGlobalProperty(OpenmrsConstants.GP_HOST_URL)
+//		Delete this when removing OpenmrsConstants.GP_HOST_URL global property.
```

Comment:
In the above comment, can you clarify what is to be deleted?

Reply:
is it ok if I introduced a separate method for it??

Reply:
No problem at all.

Reply:
I refactored the code with a new method and more meaningful comments.